workflow-varscan/varscan.wdl

```diff
@@ -130,9 +141,9 @@ input {
  File inputTumor
  File inputTumorIndex
  File inputNormalIndex
- String? refFasta = "$HG19_ROOT/hg19_random.fa"
- String? modules  = "samtools/0.1.19 hg19/p13"
- String? samtools = "$SAMTOOLS_ROOT/bin/samtools"
+ String refFasta = "$HG19_ROOT/hg19_random.fa"
```

Comment:
is this working only for HG19?  what if hg38 is being used?

Reply:
Shesmu for this workflow will support hg19, hg38 and mm10. If needed, we can add  more